### ISSUE TITLE
cifa10 accuracies

### ISSUE THREAD

**Issue by a-maci** [wenxinxu/ResNeXt-in-tensorflow#5]
When executed (python cifar10_train.py --version='test') with the given hyper-params, I get the following top1 accuracies:

Train top1 error =  0.0078125
Validation top1 error = 0.1641
Validation loss =  0.650083

The validation error of 16.41% is way off the 3.65% number mentioned in the paper (Table 7). 
Any idea of what is going wrong?

**Comment by a3551571** [comment]
excuse me.I met the same problem as you.Did you solve the problem?
I haved tried 4 ways to run the code

'python cifar10_train.py --version='test''
'python cifar10_train.py --version='test',num_resnext_blocks=3,cardinality=4'
'python cifar10_train.py --version='test',num_resnext_blocks=3,cardinality=8'
'python cifar10_train.py --version='test',num_resnext_blocks=2,cardinality=64'

The validation error is still about 16%
